Linux ubuntu feeds working (#21637)
dotnet/docs: docs/core/install/linux-ubuntu.md

## 20.10 ✔️

> [!IMPORTANT]

> > .NET Core 2.1 is not yet available in the package feed.

Author: ```suggestion
> .NET Core 2.1 isn't yet available in the package feed.
```
